systemtests/src/test/java/io/enmasse/systemtest/marathon/MarathonTestBase.java

```diff
@@ -310,9 +310,16 @@ void doTestCreateDeleteAddressesViaAgentLong(AddressSpace addressSpace) throws E
         ArrayList<Destination> addresses = generateQueueTopicList("via-web", IntStream.range(0, addressCount));
 
         runTestInLoop(30, () -> {
-            consoleWebPage.createAddressesWebConsole(addresses.toArray(new Destination[0]));
-            consoleWebPage.deleteAddressesWebConsole(addresses.toArray(new Destination[0]));
-            Thread.sleep(5000);
+            try {
+                consoleWebPage.createAddressesWebConsole(addresses.toArray(new Destination[0]));
+                consoleWebPage.deleteAddressesWebConsole(addresses.toArray(new Destination[0]));
+                Thread.sleep(5000);
+            } catch (Exception ex) {
+                selenium.setupDriver(environment, kubernetes, buildDriver());
+                consoleWebPage = new ConsoleWebPage(selenium, getConsoleRoute(addressSpace), addressApiClient, addressSpace, user);
+                consoleWebPage.openWebConsolePage(user);
+                throw new Exception(ex);
```

Comment:
and I suppose I left this throw exception here by mistake during my testing, I will remove it